views/index.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href='/style.css' rel='stylesheet' type='text/css'>
+</head>
+  <body>
+    <h1>Welcome to Chitter<h1>
+    <form action='/signup' method='GET'>
+      <input type='submit' value='Sign up' id='Sign up' style="float: right;">
+    </form>
+    <br><br>
+    <form action='peep' method='POST'>
+      <textarea name='new_peep' cols='50' rows='4' maxlength='150' placeholder='New Peep! Max charicter limit 150'></textarea><br>
```

Comment:
I like that you have specified a max character limit on the peep, both in the placeholder and in maxlength